frontend/config-assistant/src/components/code-editor/AceEditor.vue

```diff
@@ -74,6 +75,15 @@ onMounted(() => {
     editor.value.getSession().setMode('ace/mode/yaml');
   }
 
+  watch(
+    () => useSettingsStore().settingsData.codeFontSize,
+    newFontSize => {
+      if (editor.value) {
+        editor.value.setOptions({fontSize: newFontSize});
+      }
+    }
```

Comment:
add `{ initial: true }` as option to the watch, then it might work when you switch page

Reply:
I use watchEffect, then it works